controller/test/sensor_tests/sensors_test.cpp

```diff
@@ -32,10 +32,13 @@ limitations under the License.
 
 // Maximum allowable delta between calculated sensor readings and the input
 // pressure waveform [kPa]
-static const float COMPARISON_TOLERANCE = 0.005f;
+static const float COMPARISON_TOLERANCE_PRESSURE = 0.005f;
 
 // Maximum allowable delta between calculated and actual volumetric flow
-static const float COMPARISON_TOLERANCE_FLOW = 5.0e-5f;
+static const float COMPARISON_TOLERANCE_FLOW_CUBIC_M_PER_SEC = 5.0e-5f;
+
+// Maximum allowable delta between calculated and actual volume (= 5 ml)
```

Comment:
This comment is out of date

Reply:
yes missed it, I had to change that so the oversampling test would fail with those flows.
updated.